### ISSUE TITLE
Issue with pass-through truth

### ISSUE THREAD

**Issue by sindhu-ku** [DUNE/ND_CAFMaker#40]
There is an issue when not using GENIE files for truth-matching that needs to be looked into
```
Could not find a true interaction with ID <id> already in this StandardRecord, and you asked me not to make a new one
terminate called after throwing an instance of 'std::runtime_error
```

**Comment by chenel** [comment]
Hmm.  Could you run the cafmaker under gdb and get the stack trace where the exception is being thrown?

**Comment by chenel** [comment]
Thanks.  I think I've fixed it now (on the same branch in #41), if you don't mind having another look.  (I tested it with one input file and it ran through without crashing, at least.)
